src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -281,13 +289,47 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
     return this._loadPromise;
   }
 
+  /**
+   * handle decode error - reload the video and seek to last currentTime
+   * @returns {void}
+   */
+  handleDecodeError(): void {
```

Comment:
make this private
```suggestion
  _handleDecodeError(): void {
```

Also change the jsdoc to mention private and the call from `handleMediaError`